Taking over: Adds dependencies, making a PR, and updating documentation to NDG (#1315)
pyne/pyne: docs/install/source.rst

  --clean [CLEAN]       removes the build directory before continuing.
  --user [USER]         Installs into ~/.local

#. cmake:  CMake arguments.

   -D VAR                Set environment variable.
  --build-type BT       Set build type via CMAKE_BUILD_TYPE, e.g. Release
                          or Debug.
  --deps-root DEPS_ROOT The path to the directory containing all
                          dependencies
  --fast  (default)                Will try to compile from assembly, if possible.
                          This is faster than compiling from source.
  --slow                Will NOT try to compile from assembly, if possible.
                          This is slower as it must compile from source.

#. make:  Make arguments.

   -j J                  Degree of parallelism for build.

#. other/dependencies:

  --hdf5 HDF5           Path to HDF5 root directory.
  --moab [MOAB]         Path to MOAB root directory.
  --dagmc [DAGMC]       Path to DAGMC root directory.
  --prefix PREFIX       Prefix for install location.
  --build-dir BLD_DIR   where to place the build directory
  --bootstrap           Bootstraps the PyNE installation, including

>                                nuc_data_make and possibly decaygen.

Reviewer: ```suggestion
#. optional arguments:

  :code:`-h`, :code:`--help`: show this help message and exit

  :code:`--clean`: removes the build directory before continuing.

  :code:`--user`: Installs into :code:`~/.local`

#. cmake:  CMake arguments.

  :code:`-D VAR`: Set environment variable.

  :code:`--build-type BT`: Set build type via :code:`CMAKE_BUILD_TYPE`, e.g. "release"  or "debug".

  :code:`--deps-root DEPS_ROOT`: The path to the directory containing all dependencies

  :code:`--fast`: (default) Will try to compile from assembly, if possible. This is faster than compiling from source.

  :code:`--slow`: Will NOT try to compile from assembly, if possible. This is slower as it must compile from source.

#. make:  Make arguments.

  :code:`-j J`: Degree of parallelism for build.

#. other/dependencies:

  :code:`--hdf5 HDF5_PATH`: Path to HDF5 root direcitory.

  :code:`--moab MOAB_PATH`: Path to MOAB root directory.

  :code:`--dagmc DAGMC_PATH`: Path to DAGMC root directory.

  :code:`--prefix INSTALL_PATH`: Prefix for install location.

  :code:`--build-dir BUILD_DIR`: where to place the build directory

  :code:`--bootstrap`: Bootstraps the PyNE installation, including :code:`nuc_data_make` and possibly decaygen.

```

Reviewer: I fixed it. This is how it renders now:
![Screenshot from 2020-08-19 13-49-27](https://user-images.githubusercontent.com/4060017/90677485-0db63880-e223-11ea-8355-95d95a44633e.png)

Reviewer: Oh- except number is messed up. It needs correct enumeration in the list now

.. _source:

==============
Source Install 
==============

------------
Dependencies
------------
PyNE has the following dependencies:

   #. `Fortran compiler <https://gcc.gnu.org/wiki/GFortran>`_
   #. `C++ compiler <https://gcc.gnu.org/>`_
   #. `CMake <http://www.cmake.org/>`_ (>= 2.8.5)
   #. `NumPy <http://www.numpy.org/>`_ (>= 1.8.0)
   #. `SciPy <http://www.scipy.org/>`_
   #. `Cython <http://cython.org/>`_ (>= 0.19.1)
   #. `HDF5 <http://www.hdfgroup.org/HDF5/>`_
   #. `PyTables <http://www.pytables.org/>`_
   #. `Python <http://www.python.org/>`_
   #. `LAPACK <http://www.netlib.org/lapack/>`_
   #. `BLAS <http://www.netlib.org/blas/>`_
   #. `Jinja2 <http://jinja.pocoo.org/>`_

Optional Depenendencies:
   #. `MOAB <https://press3.mcs.anl.gov/sigma/moab-library>`_
   #. `DAGMC <https://svalinn.github.io/DAGMC/install/index.html>`__
   #. `OpenMC <https://docs.openmc.org/en/stable/quickinstall.html>`_
   
To run tutorial and examples:
   #. `jupyter <http://jupyter.org/>`_

Many, if not all of these dependencies can be installed using a package manager

> (i.e. `apt-get` on Linux, `MacPorts <https://www.macports.org/>`__ or `Homebrew

Reviewer: ```suggestion
(i.e. :code:`apt-get` on Linux, `MacPorts <https://www.macports.org/>`__ or `Homebrew
```

   #. `C++ compiler <https://gcc.gnu.org/>`_
   #. `CMake <http://www.cmake.org/>`_ (>= 2.8.5)
   #. `NumPy <http://www.numpy.org/>`_ (>= 1.8.0)
   #. `SciPy <http://www.scipy.org/>`_
   #. `Cython <http://cython.org/>`_ (>= 0.19.1)
   #. `HDF5 <http://www.hdfgroup.org/HDF5/>`_
   #. `PyTables <http://www.pytables.org/>`_
   #. `Python <http://www.python.org/>`_
   #. `LAPACK <http://www.netlib.org/lapack/>`_
   #. `BLAS <http://www.netlib.org/blas/>`_
   #. `Jinja2 <http://jinja.pocoo.org/>`_

Optional Depenendencies:
   #. `MOAB <https://press3.mcs.anl.gov/sigma/moab-library>`_
   #. `DAGMC <https://svalinn.github.io/DAGMC/install/index.html>`__
   #. `OpenMC <https://docs.openmc.org/en/stable/quickinstall.html>`_
   
To run tutorial and examples:
   #. `jupyter <http://jupyter.org/>`_

Many, if not all of these dependencies can be installed using a package manager
(i.e. :code:`apt-get` on Linux, `MacPorts <https://www.macports.org/>`__ or `Homebrew
<https://brew.sh/>`__ on MacOS, or `Anaconda <https://www.anaconda.com/>`__ on both, etc.)

Most common install flags:
**************************
The list of the possible flags can be retrieved using:

  python setup.py --help

> #. optional arguments:
> .. hlist::
>     :columns: 1
>   -h, --help            show this help message and exit
>   --clean [CLEAN]       removes the build directory before continuing.
>   --user [USER]         Installs into ~/.local
> 
> 
> #. cmake:  CMake arguments.
> 
> .. hlist::
>     :columns: 1
> 
>   -D VAR                Set environment variable.
>   --build-type BT       Set build type via CMAKE_BUILD_TYPE, e.g. Release
>                           or Debug.
>   --deps-root DEPS_ROOT The path to the directory containing all
>                           dependencies
>   --fast  (default)                Will try to compile from assembly, if possible.
>                           This is faster than compiling from source.
>   --slow                Will NOT try to compile from assembly, if possible.
>                           This is slower as it must compile from source.
> 
> 
> #. make:  Make arguments.
> 
> .. hlist::
>     :columns: 1
> 
>    -j J                  Degree of parallelism for build.
> 
> 
> #. other/dependencies:
> 
> .. hlist::
>     :columns: 1
> 
>   --hdf5 HDF5           Path to HDF5 root directory.
>   --moab [MOAB]         Path to MOAB root directory.
>   --dagmc [DAGMC]       Path to DAGMC root directory.
>   --prefix PREFIX       Prefix for install location.
>   --build-dir BLD_DIR   where to place the build directory
>   --bootstrap           Bootstraps the PyNE installation, including
>                                nuc_data_make and possibly decaygen.

Reviewer: ```suggestion
#. optional arguments:

  :code:`-h`, :code:`--help`: show this help message and exit

  :code:`--clean`: removes the build directory before continuing.

  :code:`--user`: Installs into :code:`~/.local`

#. cmake:  CMake arguments.

  :code:`-D VAR`: Set environment variable.

  :code:`--build-type BT`: Set build type via :code:`CMAKE_BUILD_TYPE`, e.g. "release"  or "debug".

  :code:`--deps-root DEPS_ROOT`: The path to the directory containing all dependencies

  :code:`--fast`: (default) Will try to compile from assembly, if possible. This is faster than compiling from source.

  :code:`--slow`: Will NOT try to compile from assembly, if possible. This is slower as it must compile from source.

#. make:  Make arguments.

  :code:`-j J`: Degree of parallelism for build.

#. other/dependencies:

  :code:`--hdf5 HDF5_PATH`: Path to HDF5 root direcitory.

  :code:`--moab MOAB_PATH`: Path to MOAB root directory.

  :code:`--dagmc DAGMC_PATH`: Path to DAGMC root directory.

  :code:`--prefix INSTALL_PATH`: Prefix for install location.

  :code:`--build-dir BUILD_DIR`: where to place the build directory

  :code:`--bootstrap`: Bootstraps the PyNE installation, including :code:`nuc_data_make` and possibly decaygen.

```

Reviewer: numbering on the enumeration is still messed up though- also this removes column formatting (I wasn't sure how to make that work, but looks like maybe you got it?)
This is how this renders:
![Screenshot from 2020-08-19 13-49-27](https://user-images.githubusercontent.com/4060017/90677913-a482f500-e223-11ea-987a-a70203f1279d.png)

Optional Depenendencies:
   #. `MOAB <https://press3.mcs.anl.gov/sigma/moab-library>`_
   #. `DAGMC <https://svalinn.github.io/DAGMC/install/index.html>`__
   #. `OpenMC <https://docs.openmc.org/en/stable/quickinstall.html>`_
   
To run tutorial and examples:
   #. `jupyter <http://jupyter.org/>`_

Many, if not all of these dependencies can be installed using a package manager
(i.e. `apt-get` on Linux, `MacPorts <https://www.macports.org/>`__ or `Homebrew
<https://brew.sh/>`__ on MacOS, or `Anaconda

>  <https://www.anaconda.com/>`__ on both, etc.)

Reviewer: this line being broken into two separate lines is causing rendering issues with the anaconda hyperlink
```suggestion
<https://brew.sh/>`__ on MacOS, or `Anaconda <https://www.anaconda.com/>`__ on both, etc.)
```

>  13 <https://www.anaconda.com/>`__ on both, etc.)

Reviewer 2: ```suggestion
 <https://www.anaconda.com/>`__ on both, etc.)
```

Most common install flags:
**************************
The list of the possible flags can be retrieved using:

> unzipped directory::

Reviewer: I don't understand this line and what it is trying to say

Author: `unzipped directory` is supposed to be A RST file keyword I believe

does it not render correctly on build ?

Reviewer: ![Screenshot from 2020-08-19 13-10-55](https://user-images.githubusercontent.com/4060017/90673483-6f73a400-e21d-11ea-8ce9-2c722793da8f.png)

>   python setup.py --help

Reviewer: ```suggestion
unzipped directory::

  python setup.py --help
```

Reviewer: need a blank line to render as code

#. optional arguments:
  -h, --help            show this help message and exit
  --clean [CLEAN]       removes the build directory before continuing.
  --user [USER]         Installs into ~/.local

#. cmake:  CMake arguments.

   -D VAR                Set environment variable.
  --build-type BT       Set build type via CMAKE_BUILD_TYPE, e.g. Release
                          or Debug.
  --deps-root DEPS_ROOT The path to the directory containing all
                          dependencies
  --fast  (default)                Will try to compile from assembly, if possible.
                          This is faster than compiling from source.
  --slow                Will NOT try to compile from assembly, if possible.
                          This is slower as it must compile from source.

#. make:  Make arguments.

   -j J                  Degree of parallelism for build.

#. other/dependencies:

>    --hdf5 HDF5           Path to HDF5 root directory.

Reviewer 3: ```suggestion
  --hdf5 HDF5           Path to HDF5 root directory.
```

  --moab [MOAB]         Path to MOAB root directory.
  --dagmc [DAGMC]       Path to DAGMC root directory.
  --prefix PREFIX       Prefix for install location.
  --build-dir BLD_DIR   where to place the build directory
  --bootstrap           Bootstraps the PyNE installation, including

>                                nuc_data_make and possibly decaygen.

Reviewer: formatting in this entire section is messed up- I need to look at how to do it properly before I can make a suggested change

Reviewer: I will come back to this section with a PR later

Reviewer: FYI- this is how this currently renders and I am actually struggling to get it rendered properly
![Screenshot from 2020-08-19 13-28-12](https://user-images.githubusercontent.com/4060017/90675346-f164cc80-e21f-11ea-87c0-f48a934deda2.png)

Author: removed it
